### ISSUE TITLE
Easy to follow documentation?

### ISSUE THREAD

**Issue by bokowski** [desaster/pdp2011#1]
Hi, thank you for making your work available on GitHub!

It would be great if you could add a bit of information on how to program the Cyc1000. I found this project for programming a Cyc1000 without having to install Quartus: https://github.com/trabucayre/cycloader - it seems that you just need to include the generated file in the Git repository so that programming the Cyc1000 is a one-liner (well, on Linux, after compiling that tool...).

After programming the Cyc1000, how would I put the right "stuff" on the Micro SD card if I'm using your little adapter board for the PiDP-11? After that, I would imagine that I can figure out the rest. :-)

**Comment by desaster** [comment]
Hi,

The PDP2011 is by Sytse van Slooten (not me), and my github project only holds my changes to support the CYC1000 board.

I think for general PDP2011 questions, like what to put on the sdcard, I should point towards the author and the official site at http://pdp2011.sytse.net/. I've created my RSX-11M system similar to these instructions http://pdp2011.sytse.net/wordpress/pdp-11/sessions/rsx-11m-plus/

I could add some instructions for programming the CYC1000. For now I'll point you to this forum post:

https://forum.trenz-electronic.de/index.php/topic,1050.msg4762.html#msg4762

So basically:
1) compile the project in quartus
2) follow the forum instructions to flash the board

I can add a pre-compiled file so you can skip step 1 in a week or so. However, there are some compile time options that might be different depending on what features you want;
* The disk controller to use
* Which serial port to use (builtin or the CP2102 on the adapter board)

For documentation on PDP2011 configuration, see http://pdp2011.sytse.net/wordpress/howto/configuration/

I hope this helps!

**Comment by desaster** [comment]
I found this .jic file on my laptop, I'm not 100% sure what options it was built with, but I assume RP06 (for RSX-11M+) and the CP2102 serial output (on the adapter board).

I'm also not sure which bootloader it's using, the one by Sytse or the real one that will ask for a boot device.

[cons-cyc1k-jic-sof.zip](https://github.com/desaster/pdp2011/files/3998523/cons-cyc1k-jic-sof.zip)
